### ISSUE TITLE
SQLite: Client-eval OrderBy() on DateTimeOffset, decimal, TimeSpan & ulong

### ISSUE THREAD

**Issue by bricelam** [aspnet/EntityFrameworkCore#11925]
#11889 updates `<`, `<=`, `>`, `>=`, MIN & MAX to client-eval on these types. Should we do the same for `ORDER BY`?

**Comment by smitpatel** [comment]
Possible duplicate of #10265

**Comment by bricelam** [comment]
@smitpatel Are Average, Max, Min, and Sum regressed in 3.0? If not, I think we can punt this.

**Comment by smitpatel** [comment]
Aggregate are not blocked right now. It can be done once we refactor for GroupBy aggregate support.

**Comment by divega** [comment]
Clearing up milestone so we can decide what to do in triage.

**Comment by Ignatella** [comment]
And TimeSpan OrderBy still does not work

**Comment by ajcvickers** [comment]
@Ignatella Please file a new issue and attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing so that we can investigate.

**Comment by bricelam** [comment]
@Ignatella After #18844, you could do `.OrderBy(e => e.TimeSpan.TotalDays)`
